llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -716,6 +716,28 @@ RISCVTTIImpl::getMaskedMemoryOpCost(unsigned Opcode, Type *Src, Align Alignment,
   return getMemoryOpCost(Opcode, Src, Alignment, AddressSpace, CostKind);
 }
 
+static bool hasOptimizedSegmentLoadStore(unsigned NF,
+                                         const RISCVSubtarget *ST) {
+  switch (NF) {
+  case 2:
```

Comment:
Save some lines via using macros?

Reply:
I considered it but remembered coming across some PRs where people were actively trying to remove uses of macros in LLVM, e.g. https://github.com/llvm/llvm-project/pull/105551

I'm not strongly opinionated on this, is there a general stance we should take here?

Reply:
> is there a general stance we should take here?

Not found in https://llvm.org/docs/CodingStandards.html. I think it's OK not to use macros here, just a weak suggestion.